casestudies/load-testing/README.md

```diff
@@ -0,0 +1,70 @@
+# ACK Performance at Scale
+
+In this usecase, we would deploy 10K SNS Topics, 20K SQS Queues and 20K SNS Subscriptions i.e 50K resources at a time using SQS and SNS ACK Controllers to examine performance of ACK and other metrics.
+
+**Testing environment:**
+* EKS cluster
+* ACK SNS and AWS SQS Controller
+* Prometheus + Grafana Stack
+
+![](ack.drawio.png)
+
+Prometheus is configured to scrape ACK Controller metrics which will help us analyze the performance and other metrics like resource counts, reconciliation time etc.
+*The 50K resources would be deployed using a Helm Chart.*
+
+After installing the Helm chart, following were the observations:
+
+* <u>**Analysing ACK Controller's resources behaviour**</u>
+The default installation of SQS and SNS ACK Controllers via Helm Chart configures deployment's CPU and Memory resource limits to 100m and 128Mi respectively. On deploying 50K resources, both SQS and SNS Controllers pods got CRASHED with OOM Killed error.
+
+
+![](resource_limits_breach.png)
+
+On analysing the pods runtime CPU and Memory resources in Grafana, the controller’s runtime CPU requests were more than the default controller’s CPU Limits (100m), whereas the controllers memory were requesting approximately 400m of CPU and the controller’s default memory limit was configured to be 128Mi, but at the runtime the controller were requesting for approximately 1Gi.
+
+We then upgraded ACK Controllers CPU and memory resource limits to 500m and 1Gi.
```

Comment:
Another subtitle "Test 2: CPU 500m and Memory 1Gi"

Reply:
Updated the same